packages/apps/eval-cloudflare-worker/src/404.ts

```diff
@@ -0,0 +1 @@
+export default new Response('route could not be found', { status: 404 });
```

Comment:
might need to use this for all the other requests to clouldflare currently not using it